.github/workflows/gradle-build.yml

```diff
@@ -32,9 +32,9 @@ jobs:
     - name: Validate gradle wrapper
       uses: gradle/wrapper-validation-action@v1
     - name: Build
-      uses: gradle/gradle-build-action@v2.4.2
+      uses: gradle/gradle-build-action@v2.7.1
       with:
-        arguments: build
+        arguments: build -Porg.gradle.jvmargs=-Xmx2048m
```

Comment:
There was an OOME again (and I had to cancel the build - see https://github.com/tomorrow-one/transactional-outbox/actions/runs/5996501158/attempts/1 ), so I'm not sure if this is really working / doing what's expected.